src/helper-modules/companies-house/wrap-companies-house-http-error-callback.ts

```diff
@@ -9,21 +9,25 @@ type CompaniesHouseHttpErrorCallback = (error: Error) => ObservableInput<never>;
 export const createWrapCompaniesHouseHttpGetErrorCallback =
   ({ messageForUnknownError, knownErrors }: { messageForUnknownError: string; knownErrors: KnownErrors }): CompaniesHouseHttpErrorCallback =>
   (error: Error) => {
-    if (error instanceof AxiosError && error.response && typeof error.response.data === 'object') {
-      const errorResponseData = error.response.data;
-      let errorMessage: string;
+    if (error instanceof AxiosError) {
+      let message: string;
+      const status: number = error?.response?.status;
```

Comment:
`const status: number | null= error?.response?.status;`.... oddly 

https://github.com/axios/axios/blob/8966ee7ea62ecbd6cfb39a905939bcdab5cf6388/lib/core/AxiosError.js#L50

Reply:
Have removed as this might not be necessary anymore.